packages/botonic-core/src/hubtype-service.js

```diff
@@ -86,4 +86,10 @@ export class HubtypeService {
       }
     )
   }
+
+  static async getWebchatVisibility({ appId }) {
+    return axios.get(
+      `${HUBTYPE_API_URL}/v1/provider_accounts/${appId}/get_webchat_visibility/`
```

Comment:
No need to put the html verb in the path: `/get_webchat_visibility/` => `/visibility/`

Reply:
sure, will be cleaner, both applied 👍